scripts/build/plugins

```diff
@@ -15,5 +15,5 @@ for p in cli-plugins/examples/* "$@" ; do
     mkdir -p "$(dirname "${TARGET_PLUGIN}")"
 
     echo "Building $GO_LINKMODE $(basename "${TARGET_PLUGIN}")"
-    (set -x ; CGO_ENABLED=0 GO111MODULE=auto go build -o "${TARGET_PLUGIN}" -tags "${GO_BUILDTAGS}" -ldflags "${GO_LDFLAGS}" ${GO_BUILDMODE} "github.com/docker/cli/${p}")
+    (set -x ; CGO_ENABLED=0 GO111MODULE=auto go build -o "${TARGET_PLUGIN}" -tags "${GO_BUILDTAGS}" -ldflags "${GO_LDFLAGS}" "github.com/docker/cli/${p}")
```

Comment:
Heh, so this fixed the plugins being compiled, but looks like the binary itself does still require `gcc`, not just `libgcc` ?

```
 > [build 2/2] RUN --mount=type=bind,target=.,ro     --mount=type=cache,target=/root/.cache     --mount=type=tmpfs,target=cli/winresources     xx-go --wrap &&     TARGET=/out ./scripts/build/binary &&     xx-verify $([ "static" = "static" ] && echo "--static") /out/docker:
0.113 + go build -o /out/docker-linux-arm-v6 -tags ' osusergo pkcs11' -ldflags ' -X "github.com/docker/cli/cli/version.GitCommit=7e15277994" -X "github.com/docker/cli/cli/version.BuildTime=2024-09-02T14:39:44Z" -X "github.com/docker/cli/cli/version.Version=pr-5388" -extldflags -static' '-buildmode=pie' github.com/docker/cli/cmd/docker
29.74 # github.com/docker/cli/cmd/docker
29.74 /usr/local/go/pkg/tool/linux_amd64/link: running armv6-alpine-linux-musleabihf-clang failed: exit status 1
29.74 clang: warning: argument unused during compilation: '-pie' [-Wunused-command-line-argument]
29.74 ld.lld: error: cannot open crtbeginT.o: No such file or directory
29.74 ld.lld: error: unable to find library -lgcc
29.74 ld.lld: error: unable to find library -lgcc_eh
29.74 ld.lld: error: cannot open crtend.o: No such file or directory
29.74 clang: error: linker command failed with exit code 1 (use -v to see invocation)
```